dev-util/statifier/statifier-1.7.4.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Statifier is a tool for creating portable, self-containing Linux executables."
+HOMEPAGE="http://statifier.sourceforge.net"
+SRC_URI="http://sourceforge.net/projects/${PN}/files/${PN}/${PV}/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+LICENSE="GPL-2"
+IUSE=""
+
+RDEPEND="app-shells/bash
+	sys-apps/coreutils
+	virtual/awk"
+
+src_prepare() {
+	# Respect users CFLAGS and LDFLAGS
+	sed -i -e "s/-Wall -O2/${CFLAGS} ${LDFLAGS}/g" "${S}/src/Makefile"
+
+	# Apply user patches
+	eapply_user
+}
+
+src_configure() {
+	# Fix permissions, as configure is not marked executable
+	fperms 755 configure
```

Comment:
`fperms` is supposed to work purely on `"${D}"`. Use `chmod` here, and prefer symbolic modes (`+x`) over forced numerics.

Reply:
Okay, will change it. May I ask, why fperms is only for "${D}"? It apparently worked in that case?

Reply:
That's by design. The whole point of existence of `fperms` is to have a wrapper that prepends D. It 'apparently working' is just a lazy implementation in Portage.